### PR TITLE
Feat: toast component

### PR DESCRIPTION
**Pull request by vivekmehto**
https://user-images.githubusercontent.com/76930847/152692017-f20912ea-ae02-4a04-a06d-edca3a08aeda.mov

### CODE REVIEW

**Review by simrk — COMMENTED** [review]
Lgtm

**Review by shuntzhere — COMMENTED** [review]
Check for display none with justify content.

File: components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.toast {
+  margin: 0.8rem;
+  padding: 1rem;
+  position: fixed;
+  bottom: 0;
+  border-radius: 5px;
+  background-color: var(--grey);
+  display: none;
+  justify-content: center;
+  align-items: center;
```
Comment (shuntzhere) [inline components/toast/toast.css:10]: I think justify-content is only applicable for flex containers. You might want to check this once.

Comment (vivekmehto) [reply components/toast/toast.css:10]: yes, it works with flex. I am using js to toggle display to flex.

Comment (shuntzhere) [reply components/toast/toast.css:10]: Ah got it.

**Review by shuntzhere — APPROVED** [review]

**Pull request merged**